Make failed on OracleLinux8 box

make failed on OracleLinux8 box like this (CentOS7 fails similar problem)

```
git clone git@github.com:yoku0825/mysqlredo.git
cd mysqlredo/
cmake -DDOWNLOAD_BOOST=1 -DWITH_BOOST=./boost -DFORCE_INSOURCE_BUILD=1 .
make mysqlredo
```

Output

```
[100%] Linking CXX executable ../runtime_output_directory/mysqlredo
/opt/rh/gcc-toolset-11/root/usr/bin/ld: ../sql/libsql_main.a(mysqld.cc.o): in function `component_infrastructure_init()':
/home/yoku0825/mysql/src/git/mysqlredo/sql/mysqld.cc:2048: undefined reference to `mysql_service_mysql_rwlock_v1'
/opt/rh/gcc-toolset-11/root/usr/bin/ld: /home/yoku0825/mysql/src/git/mysqlredo/sql/mysqld.cc:2057: undefined reference to `mysql_service_mysql_psi_system_v1'
/opt/rh/gcc-toolset-11/root/usr/bin/ld: /home/yoku0825/mysql/src/git/mysqlredo/sql/mysqld.cc:2068: undefined reference to `mysql_service_mysql_runtime_error'
/opt/rh/gcc-toolset-11/root/usr/bin/ld: ../sql/server_component/libmysql_server_component_services.a(component_sys_var_service.cc.o): in function `mysql_component_sys_variable_imp::get_variable(char const*, char const*, void**, unsigned long*)':
/home/yoku0825/mysql/src/git/mysqlredo/sql/server_component/component_sys_var_service.cc:493: undefined reference to `mysql_components_handle_std_exception(char const*)'
/opt/rh/gcc-toolset-11/root/usr/bin/ld: ../sql/server_component/libmysql_server_component_services.a(component_sys_var_service.cc.o): in function `mysql_component_sys_variable_imp::unregister_variable(char const*, char const*)':
/home/yoku0825/mysql/src/git/mysqlredo/sql/server_component/component_sys_var_service.cc:551: undefined reference to `mysql_components_handle_std_exception(char const*)'
/opt/rh/gcc-toolset-11/root/usr/bin/ld: ../sql/server_component/libmysql_server_component_services.a(component_sys_var_service.cc.o): in function `mysql_component_sys_variable_imp::register_variable(char const*, char const*, int, char const*, int (*)(THD*, SYS_VAR*, void*, st_mysql_value*), void (*)(THD*, SYS_VAR*, void*, void const*), void*, void*)':
/home/yoku0825/mysql/src/git/mysqlredo/sql/server_component/component_sys_var_service.cc:399: undefined reference to `mysql_components_handle_std_exception(char const*)'
/opt/rh/gcc-toolset-11/root/usr/bin/ld: ../sql/server_component/libmysql_server_component_services.a(persistent_dynamic_loader.cc.o): in function `mysql_persistent_dynamic_loader_imp::init(void*)':
/home/yoku0825/mysql/src/git/mysqlredo/sql/server_component/persistent_dynamic_loader.cc:315: undefined reference to `mysql_components_handle_std_exception(char const*)'
/opt/rh/gcc-toolset-11/root/usr/bin/ld: ../sql/server_component/libmysql_server_component_services.a(persistent_dynamic_loader.cc.o): in function `mysql_persistent_dynamic_loader_imp::load(void*, char const**, int)':
/home/yoku0825/mysql/src/git/mysqlredo/sql/server_component/persistent_dynamic_loader.cc:433: undefined reference to `mysql_components_handle_std_exception(char const*)'
/opt/rh/gcc-toolset-11/root/usr/bin/ld: ../sql/server_component/libmysql_server_component_services.a(persistent_dynamic_loader.cc.o):/home/yoku0825/mysql/src/git/mysqlredo/sql/server_component/persistent_dynamic_loader.cc:542: more undefined references to `mysql_components_handle_std_exception(char const*)' follow
/opt/rh/gcc-toolset-11/root/usr/bin/ld: ../sql/server_component/libmysql_server_component_services.a(server_component.cc.o): in function `initialize_minimal_chassis(s_mysql_registry**)':
/home/yoku0825/mysql/src/git/mysqlredo/sql/server_component/server_component.cc:750: undefined reference to `minimal_chassis_init(s_mysql_registry**, mysql_component_t*)'
/opt/rh/gcc-toolset-11/root/usr/bin/ld: ../sql/server_component/libmysql_server_component_services.a(server_component.cc.o): in function `deinitialize_minimal_chassis(s_mysql_registry*)':
/home/yoku0825/mysql/src/git/mysqlredo/sql/server_component/server_component.cc:761: undefined reference to `minimal_chassis_deinit(s_mysql_registry*, mysql_component_t*)'
/opt/rh/gcc-toolset-11/root/usr/bin/ld: ../sql/server_component/libmysql_server_component_services.a(system_variable_source_imp.cc.o): in function `mysql_system_variable_source_imp::get(char const*, unsigned int, enum_variable_source*)':
/home/yoku0825/mysql/src/git/mysqlredo/sql/server_component/system_variable_source_imp.cc:45: undefined reference to `mysql_components_handle_std_exception(char const*)'
/opt/rh/gcc-toolset-11/root/usr/bin/ld: ../sql/server_component/libmysql_server_component_services.a(mysql_system_variable_update_imp.cc.o): in function `mysql_system_variable_update_imp::set_string(THD*, char const*, my_h_string_imp*, my_h_string_imp*, my_h_string_imp*)':
/home/yoku0825/mysql/src/git/mysqlredo/sql/server_component/mysql_system_variable_update_imp.cc:243: undefined reference to `mysql_components_handle_std_exception(char const*)'
/opt/rh/gcc-toolset-11/root/usr/bin/ld: ../sql/server_component/libmysql_server_component_services.a(mysql_system_variable_update_imp.cc.o): in function `mysql_system_variable_update_imp::set_signed(THD*, char const*, my_h_string_imp*, my_h_string_imp*, long long)':
/home/yoku0825/mysql/src/git/mysqlredo/sql/server_component/mysql_system_variable_update_imp.cc:294: undefined reference to `mysql_components_handle_std_exception(char const*)'
/opt/rh/gcc-toolset-11/root/usr/bin/ld: ../sql/server_component/libmysql_server_component_services.a(mysql_system_variable_update_imp.cc.o): in function `mysql_system_variable_update_imp::set_unsigned(THD*, char const*, my_h_string_imp*, my_h_string_imp*, unsigned long long)':
/home/yoku0825/mysql/src/git/mysqlredo/sql/server_component/mysql_system_variable_update_imp.cc:338: undefined reference to `mysql_components_handle_std_exception(char const*)'
/opt/rh/gcc-toolset-11/root/usr/bin/ld: ../sql/server_component/libmysql_server_component_services.a(mysql_system_variable_update_imp.cc.o): in function `mysql_system_variable_update_imp::set_default(THD*, char const*, my_h_string_imp*, my_h_string_imp*)':
/home/yoku0825/mysql/src/git/mysqlredo/sql/server_component/mysql_system_variable_update_imp.cc:378: undefined reference to `mysql_components_handle_std_exception(char const*)'
/opt/rh/gcc-toolset-11/root/usr/bin/ld: ../sql/server_component/libmysql_server_component_services.a(mysql_thd_attributes_imp.cc.o):/home/yoku0825/mysql/src/git/mysqlredo/sql/server_component/mysql_thd_attributes_imp.cc:58: more undefined references to `mysql_components_handle_std_exception(char const*)' follow
collect2: error: ld returned 1 exit status
make[3]: *** [client/CMakeFiles/mysqlredo.dir/build.make:224: runtime_output_directory/mysqlredo] Error 1
make[2]: *** [CMakeFiles/Makefile2:30579: client/CMakeFiles/mysqlredo.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:30586: client/CMakeFiles/mysqlredo.dir/rule] Error 2
```

--- a/client/CMakeLists.txt
+++ b/client/CMakeLists.txt
@@ -102,7 +102,7 @@ MYSQL_ADD_EXECUTABLE(mysqlredo
   mylog0log.cc
   mymtr0log.cpp
   INCLUDE_DIRECTORIES ../storage/innobase/include ../storage/innobase
-  LINK_LIBRARIES innobase minchassis # mysqlservices sql_main sql_dd mysys
+  LINK_LIBRARIES innobase mysql_server_component_services minchassis # mysqlservices sql_main sql_dd mysys
 )
 ADD_DEPENDENCIES(mysqlredo minchassis)
 ADD_DEPENDENCIES(mysqlredo innobase minchassis)

